failed when deploy, can you please fix it

<details><summary>Vercel Console</summary>

<pre>
```
> lumos-website@0.0.0 build
--
20:32:27.077 | > docusaurus build
20:32:27.077 |  
20:32:28.861 | [INFO] [en] Creating an optimized production build...
20:32:29.631 | undefined
20:32:29.634 | node16.16
20:32:29.763 | [ERROR] Unable to build website for locale en.
20:32:29.763 | [ERROR] BrowserslistError: Unknown browser query `modern`. Maybe you are using old Browserslist or made typo in query.
20:32:29.764 | at unknownQuery (/vercel/path0/website/node_modules/browserslist/index.js:283:10)
20:32:29.764 | at Function.select (/vercel/path0/website/node_modules/browserslist/index.js:1149:15)
20:32:29.764 | at /vercel/path0/website/node_modules/browserslist/index.js:314:29
20:32:29.764 | at Array.reduce (<anonymous>)
20:32:29.764 | at resolve (/vercel/path0/website/node_modules/browserslist/index.js:304:34)
20:32:29.765 | at browserslist (/vercel/path0/website/node_modules/browserslist/index.js:406:21)
20:32:29.765 | at Object.load (/vercel/path0/website/node_modules/webpack/lib/config/browserslistTargetHandler.js:70:9)
20:32:29.765 | at TARGETS.web (/vercel/path0/website/node_modules/webpack/lib/config/target.js:88:47)
20:32:29.765 | at getTargetProperties (/vercel/path0/website/node_modules/webpack/lib/config/target.js:296:19)
20:32:29.765 | at applyWebpackOptionsDefaults (/vercel/path0/website/node_modules/webpack/lib/config/defaults.js:141:6)
20:32:29.798 | Error: Command "npm run build-lumos && npm run build" exited with 1
```
</pre>

</details>

--- a/website/docusaurus.config.js
+++ b/website/docusaurus.config.js
@@ -63,10 +63,8 @@ const config = {
     ],
     () => ({
       name: "node-polyfill",
-      configureWebpack(config) {
-        console.log(config.target);
+      configureWebpack() {
         return {
-          target: 'browserslist:modern',
           resolve: {
             fallback: {
               crypto: require.resolve("crypto-browserify"),

